config.go

```diff
@@ -15,7 +14,7 @@ type Config struct {
 	Wait             *Wait
 	NotifyCmd        string
 	NotifyOutput     bool
-	NotifyContainers map[string]docker.Signal
+	NotifyContainers map[string]int
```

Comment:
Why did you use int instead of docker.Signal? I thought docker.Signal was needed to validate the signal is a valid signal.